atm.js

```diff
@@ -1,21 +1,57 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+//function gaATM with OOP
+function gaATM() {
+  this.balance = 0;
+  this.checking = {
+    input: $("#newChecking"),
+    total: 0
+  };
+  this.savings = {
+    input: $("#newSavings"),
+    total: 0
+  };
 }
 
-function checkingWithdrawal(amount){
-
+gaATM.prototype.balance = function() {
+  this.gaBalance = this.checking.total + this.savings.total;
+  var total = this.checking.total + this.savings.total;
+  return total;
+};
+
+var bank = new gaATM();
+var checkingBalance = $("#checkingBalance").addClass("empty");
+var savingsBalance = $("#savingsBalance").addClass("empty");
+
+//NEEDED EVENT LISTENERS
+
+//checking deposit
+//the input is depositCash, and we are creating an on click function event
+$("input.depositCash").on("click", function(event) {
+//prevent default behavior
+  event.preventDefault();
+//ensures user input is an integer
+  var input = parseInt(bank.checking.input.val());
+  checkingDeposit(input);
+//clear the input by utilizing an empty string
+  bank.checking.input.val("");
 }
-
-function savingsDeposit(amount){
-
+};
```

Comment:
From debugging in the console, there is a syntax error here on line 36 & 37, it should be `});` instead of `}};`